### ISSUE TITLE
CIRIquant analyse single end  data

### ISSUE THREAD

**Issue by 1409605824** [Kevinzjy/CIRIquant-cookbook#4]
Hello，can CIRIquant be used to analyse the single end data? CIRI2 can analyse single or paired end data,but when i want to do differential expression of cir-RNA using CIRIquant, I find the CIRIquant just can be input paired data because of the " -1 -2 "parameter, so can you test me how to deal with it, or should i use other tools? For example, the other tools "sailfish-as‘’ can be
used for single or paired data，because its parameter " -1  -2 for paried end data (*.fastq.gz) , and -r for single end data(*.fastq.gz), but i don't find it in CIRIquant parameter.
THANKS

**Comment by Kevinzjy** [comment]
No, the current version of CIRIquant don't support single end data as input. I would recommend you run CIRI2 and stringtie to generate read count table of gene and circular RNAs manually, and use them as input for differential expression analysis.
